packages/wallet/src/W3mFrame.ts

```diff
@@ -124,7 +124,6 @@ export class W3mFrame {
           throw new Error('W3mFrame: iframe is not set')
         }
         W3mFrameSchema.appEvent.parse(event)
-        window.postMessage(event)
         this.iframe.contentWindow.postMessage(event, '*')
```

Comment:
@zoruka Is it because the `this.iframe.contentWindow.postMessage(event, '*')` does the thing?

Reply:
yes, exactly!